docs/manual/shared_memory/developer_hints.rst

```diff
@@ -0,0 +1,49 @@
+.. include:: ../external-links.part.rst
+
+.. index:: 
+  single: Shared memory; Developer hints
+  single: Developer hints
+  single: Contributing
+
+.. _dev_hints:
+
+Developer hints
+---------------
+
+The initial implementation is from |url::adlink-ROS|. To integrate the latest 
+|url::iceoryx_link| C-API and support zero copy data transfer, contributions 
+were made by |url::Apex.AI|. Further contributions and feedback from the community 
+are very welcome, see :ref:`contributing_to_dds`.
+
+The following is a list of useful tips:
+
+* Most of the shared memory modification is under the define :c:macro:`DDS_HAS_SHM`.
+
+* To learn about the internal happenings of the |url::iceoryx_link| service, there 
+  is a useful tool from iceoryx called |url::iceoryx_introspection|:
```

Comment:
```suggestion
  is a useful tool from Iceoryx called |url::iceoryx_introspection|:
```

Reply:
iceoryx is lowercase, see https://iceoryx.io/v2.0.2/getting-started/what-is-iceoryx/